plugins/GSdx/GSState.cpp

```diff
@@ -46,7 +46,7 @@ GSState::GSState()
 	, m_frameskip(0)
 	, m_crcinited(false)
 {
-	m_nativeres = !!theApp.GetConfig("nativeres", 1);
+	m_nativeres = !!theApp.GetConfig("upscale_multiplier",1)==1 ? 1:0;
```

Comment:
The !! is evaluated before the == unless I have things seriously wrong. Wouldn't the correct line be
`theApp.GetConfig("upscale_multiplier",1)==1;`